ginivision/src/main/java/net/gini/android/vision/review/ReviewFragmentImpl.java

```diff
@@ -422,6 +427,8 @@ void onStop() {
     }
 
     void onSaveInstanceState(final Bundle outState) {
+        // Remove previously saved data from the memory cache
+        ParcelableMemoryCache.getInstance().removeEntriesWithTag(PARCELABLE_MEMORY_CACHE_TAG);
```

Comment:
I'm confused, why removing entries here?

Reply:
`onSaveInstanceState()` can be called multiple times without an `onCreate()` or `onRestoreInstanceState()` call afterwards. Clearing the cache here makes sure that there are no "dangling" entries in the cache.